src/Providers/STSimMapProvider.cs

```diff
@@ -1409,5 +1427,21 @@ private static bool CreateAgeTypeColorMap(Project project, string colorMapFilena
 
             return true;
         }
+
+        private static bool DatasheetHasRows(Scenario s, string datasheetName)
+        {
+            DataSheet ds = s.GetDataSheet(datasheetName);
+            return ds != null && ds.GetData().Rows.Count > 0;
+        }
+
+        private static bool ScenarioIsMultiRes(Scenario s)
+        {
+            return DatasheetHasRows(s, Strings.DATASHEET_TRG_NAME) && DatasheetHasRows(s, Strings.DATASHEET_SPICF_NAME);
+        }
+
+        private static bool ShouldShowMultiResolutionCriteriaNodes(Project project, DataStore store)
```

Comment:
Looks like the DataStore argument isn't being used in this function?

Reply:
Oh good catch! Thanks :)

Reply:
Fixed in 2269339.